Automattic/jetpack
# Contact Form: Update default labels

Fixes #27429

#### Changes proposed in this Pull Request:
* Updates the default labels logic to allow fields without any label

#### Other information:


#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

* Create a post and a Form block
* Choose a variation and add other fields to the Form
* Check if the fields allow "empty" labels
